hugolib/site.go

```diff
@@ -124,6 +126,11 @@ type SiteInfo struct {
 	preserveTaxonomyNames bool
 	paginationPageCount   uint64
 	Data                  *map[string]interface{}
+
+	Multilingual    bool
+	CurrentLanguage string
+	LanguagePrefix  string
+	Languages       []string
 }
```

Comment:
Consider if we need to export all of the above, in the light of the suggested API-changes above.